cleverhans/feat_advs.py

```diff
@@ -0,0 +1,290 @@
+import numpy as np
+import tensorflow as tf
+
+from cleverhans.attacks import Attack
+from cleverhans.model import Model
+
+
+def ffg(x, s_feat, g_feat, eps=0.3, ord=np.inf,
+        clip_min=None, clip_max=None, targeted=False):
+    """
+    TensorFlow implementation of the Fast Gradient Method.
```

Comment:
This docstring seems wrong; presumably this isn't the Fast Gradient Method

Reply:
Fixed in 7f5b62d and 9459f33.